src/main/java/formflow/library/file/FileValidationService.java

```diff
@@ -30,8 +30,9 @@
  */
 @Slf4j
 @Service
-public class FileTypeService {
+public class FileValidationService {
 
+  public static final int MB_IN_BYTES = 1024 * 1024;
```

Comment:
I was wondering if you thought about making this a long, since the filesizes will be long's (mostly). Don't feel strongly about it, but do worry about mixing data types in comparisons.  Java seems to do the right thing, but it's nice to keep datatypes aligned.